### ISSUE TITLE
Add Touch Screen Support for Dragging Images

### ISSUE THREAD

**Issue by br0wsa** [musama619/react-photo-editor#243]
Would it be possible to add support for touch screens to enable dragging images using touch gestures? 

Currently, only mouse events are implemented. Adding touch event handling (such as touchstart, touchmove, and touchend) would enhance the user experience on smartphones and other touch-enabled devices.

**Comment by musama619** [comment]
Good idea! I’ll look into adding touch events support to enable touch gestures and develop this.

**Comment by br0wsa** [comment]
GREAT !
